src/main/java/frc/robot/Constants.java

```diff
@@ -31,7 +31,7 @@ public enum GamePiece {
     NONE, CUBE, CONE, HUH
   }
 
-  public static final boolean OUTPUT_DEBUG_VALUES = false;
```

Comment:
FYI, this is set to true.

Reply:
the `USE_NETWORK_TABLES` was true too, fixed in latest commit